src/FieldError.php

```
@@ -69,52 +69,90 @@ public function clear(string|null $property = null): void
      * $firstErrors = $errors->get(true);
      * ```
      *
-     * @param bool $first Whether to return only the first error for each property.
+     * @param bool $first Whether to return only the first error for each field.
      *
-     * @return array Errors indexed by property name. Returns an empty array when no errors exist.
+     * @return array Errors indexed by field name. Returns an empty array when no errors exist.
      *
      * @phpstan-return array<string, array<int, string>|string>
      */
     public function get(bool $first = false): array
```

PHPMD (2.15.0)

[error] The method get has a boolean flag argument $first, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)